packages/components/pages/f-registration/src/tenants/es-ES.js

```diff
@@ -0,0 +1,62 @@
+export default {
+    locale: 'es-ES',
+    genericErrorMessage: 'No es posible crear una cuenta en este momento',
+
+    navLinks: {
+        termsAndConditions: {
+            prefix: 'Al crear una cuenta, aceptas nuestros ',
+            suffix: '.',
+            text: 'Términos y Condiciones',
+            url: '/info/terms-and-conditions'
+        },
+        privacyPolicy: {
+            prefix: 'Por favor, lee nuestra ',
+            text: 'Política de Privacidad',
+            url: '/info/privacy-policy'
+        },
+        cookiesPolicy: {
+            prefix: ' y ',
+            text: 'Política de Cookie',
+            url: '/info/cookies-policy',
+            suffix: '.'
+        },
+        login: {
+            text: '¿Ya formas parte de [Just Eat]?'
```

Comment:
Why is Just Eat in brackets?

Reply:
Good question - it's what we were provided in the translations but let me confirm this wasn't a mistake

Reply:
Not needed, have removed. Good spot!